File: backend-api/reservation-api/src/main/java/com/hcmute/reservation/model/dto/table/TableAlert.java
package com.hcmute.reservation.model.dto.table;

import lombok.AllArgsConstructor;
import lombok.Data;

@Data
@AllArgsConstructor
public class TableAlert {
    private Long tableId;
    private String alertType; // "START_BLINK" hoặc "STOP_BLINK"
}

File: desktop-staff/reservation-winforms/DTO/table/TableAlert.cs
using Newtonsoft.Json;
using System;

namespace reservation_winforms.DTO.table
{
    public class TableAlertMessage
    {
        [JsonProperty("tableId")]
        public long TableId { get; set; }

        [JsonProperty("alertType")]
        public string AlertType { get; set; } // "START_BLINK" hoặc "STOP_BLINK"
    }
}